functions-common

```diff
@@ -439,7 +455,7 @@ function is_clearlinux {
         GetOSVersion
     fi
 
-    [ "$os_CODENAME" = "clearlinux" ]
+    [ "$os_VENDOR" = "Clear_Linux_OS" ]
```

Comment:
This is the only real change, I did everything else already, or does it not work for you?